reference/supervisor.md

```diff
@@ -137,6 +138,9 @@ The `wb_supervisor_node_get_self()` function returns a handle to the
 utility function that simplifies the task of retrieving the base node without
 having to define a DEF name for it.
 
+The `wb_supervisor_node_get_selected()` function retrieves a handle to the currently selected node.
```

Comment:
Do you think that it is better to specify that the selected node refers to the selection in the scene tree?